airflow/contrib/operators/ssh_execute_operator.py

```diff
@@ -133,24 +133,21 @@ def execute(self, context):
                 logging.info("env: " + str(self.env))
 
             sp = hook.Popen(
-                ['-q', 'bash', remote_file_path],
-                stdout=subprocess.PIPE, stderr=STDOUT,
-                env=self.env)
+                    ['-q', 'bash', remote_file_path],
+                    stdout=subprocess.PIPE, stderr=STDOUT,
+                    env=self.env)
 
             self.sp = sp
-
-            logging.info("Output:")
-            line = ''
+            output = StringIO()
             for line in iter(sp.stdout.readline, b''):
-                line = line.decode().strip()
-                logging.info(line)
+                print >> output, line
```

Comment:
No, cStringIO is not available in python 3, let me replace this with six, Thanks